web-wallet/src/lib/migration/walletConnection.js

```diff
@@ -0,0 +1,63 @@
+import { createWeb3Modal, defaultWagmiConfig } from "@web3modal/wagmi";
+import {
+  disconnect,
+  getAccount,
+  getBalance,
+  reconnect,
+  watchAccount,
+} from "@wagmi/core";
+import { readable } from "svelte/store";
+import { bsc, mainnet } from "viem/chains";
+
+// Required project metadata
+const projectId = "b5303e1c8374b100fbb7f181884fef28";
+const metadata = {
+  description: "Dusk Web-Wallet",
+  icons: [],
+  name: "Dusk Migration",
+  url: "https://127.0.0.1:5173/dashboard/",
+};
+
+const chains = [mainnet, bsc];
+
+export const wagmiConfig = defaultWagmiConfig({
+  // @ts-ignore
+  chains,
+  metadata,
+  projectId,
+});
+
+const isBrowser = typeof window !== "undefined";
+if (isBrowser) {
```

Comment:
Why this check as we know we're in a browser?

Reply:
A leftover from the example. Not necessary in our case indeed.